### PR TITLE
Add fun animal display names

### PR DESCRIPTION
**Pull request by gmittal**
Now volunteers will see anonymous case display names as things like "Vague Manatee" or "Sharp Ant".

@ankitr PTAL.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -35,7 +36,6 @@ var db = firebase.database().ref("/");
 var hash = new hashid(process.env.HASH_SALT, process.env.HASH_LENGTH);
 
 
-
```
Comment (ankitr) [inline app.js:38]: Thank you :-P

File: app.js [hunk 2]
```diff
@@ -86,26 +86,28 @@ app.post('/api/addUserRequest', function (req, res) {
         }
 
         var pendingRequest = req.body;
-        pendingRequest["time_submitted"] = new Date();
+        pendingRequest["display_name"] = bandname();
+        pendingRequest["time_submitted"] = new Date().getTime();
```
Comment (ankitr) [inline app.js:90]: What's this change?

Comment (gmittal) [inline app.js:90]: It turns out that the epoch time (which was added in a previous pull request) got overwritten or merged wrong and was causing fatal errors. Changed it back.

**Comment by ankitr** [comment]
LGTM

**Pull request merged**